lambdas/functions/main.py

```diff
@@ -2,123 +2,155 @@
 import json
 import tempfile
 import os
+import logging
 
 from samplesheet_check import run_check, construct_logger
 
 from umccr_utils.samplesheet import SampleSheet
 from umccr_utils.globals import LOG_DIRECTORY
 
-def construct_body(check_status, error_message, log_path):
-    """Construct body from from information"""
-
-    # Get Log Data
-    with open(log_path, 'r') as log_file:
-        log_text = log_file.read()
-
-    # Remove the tmp values
-    os.remove(log_path)
-    body = {
-        "check_status": check_status,
-        "error_message": error_message,
-        "log_file": log_text
-    }
-    return json.dumps(body)
-
-def construct_response(status_code, body, origin):
-    """Construct response from parameter"""
-    
-    # Configuration of allowed origin
-    allowed_origin_list = [ 'https://sscheck.dev.umccr.org', 'https://sscheck.prod.umccr.org', 'https://sscheck.umccr.org' ]
-    
-    if origin in allowed_origin_list:
-        return_origin = origin
-    else:
-        return_origin = allowed_origin_list[0]
-        
-    response =  {
-        'statusCode': status_code,
-        'headers': {
-            'Access-Control-Allow-Headers': 'Content-Type',
-            'Access-Control-Allow-Origin': return_origin,
-            'Access-Control-Allow-Methods': 'OPTIONS,POST,GET',
-        },
-        'body': body
-    }
-
-    return response
-
+# Logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
 
 def lambda_handler(event, context):
+    """
+    Parameters
+    ----------
+    event : Object
+        An object of payload pass through the lambda
+    context : Object
+        [NOT_USED] a aws resource information
+
+    Return
+    ----------
+    error_message : str
+        any error message that stops the check
+
+    """
+    logger.info('Processing samplesheet check based on the following event.')
+    logger.info(json.dumps(event))
 
     try:
         origin = event["headers"]["origin"]
     except KeyError:
         origin = ""
-    
+    logger.info(f'Parsing origin header content: {origin}')
+
     auth_header = event["headers"]["Authorization"]
+    logger.info(f'Parsing auth_header header content: {auth_header}')
```

Comment:
I'd recommend, not to log auth header content here.

Reply:
Removed at commit 3c49e1c